src/components/custom/signup.tsx

```diff
@@ -1,9 +1,9 @@
 import React, { useState } from 'react'
-import { graphql, useStaticQuery, navigate } from 'gatsby'
+import { graphql, navigate, useStaticQuery } from 'gatsby'
```

Comment:
Out of curiosity is there a special reason we changing the order? 🤔

Reply:
No reason, just husky did it 😅